rpa-home/modules/ROOT/pages/automation-usecase-legacysystemdataextraction-extractfrompdf.adoc

```diff
@@ -0,0 +1,64 @@
+# Finding Information in a PDF
+// number the headings to correspond to the numbered list of steps
+:sectnums:
+
+First, you need to find the order ID in the PDF.
+
+As you can see in xref:automation-usecase-legacysystemdataextraction.adoc#confirmationletter-image[the image of the order confirmation letter], the order ID is always listed in a table next to the entry 'Order ID'. 
+// Do we need an image of that part of the letter here?
```

Comment:
I would snip just that table and add it - to avoid the "you can see" and "next to".

Reply:
This is important information for impelmenting the workflow so that a robot can understand it. A visually impaired user can't use RPA because image recognition is an important part.